css/style.css

```diff
@@ -99,15 +99,254 @@ a {
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  margin-top: 20px;
 }
 
 .networkContent {
   display: flex;
+  margin-top: 20px;
   list-style: none;
   justify-content: space-between;
   max-width: 150px;
   align-items: center;
+  gap: 10px;
 }
+
+.workSection {
+  margin: 20px;
+  position: relative;
+  top: 150px;
+}
```

Comment:
- [ ] As requested in the project requirements, kindly align the work section with a `display: grid;` property